Add git_guide.md and update it (#90)
mozilla-mobile/shared-docs: git_guide.md

> # Git guide

Reviewer: Can you link to this from the beginner's guides in `CONTRIBUTING_code`? https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING_code.md#beginners-guides

Author: provide a link in beginners-guides.

This guide is designed to help you get started with git and use its basics commands.

- [Getting started](#getting-started)
- [First contribution](#first-contribution)
- [Good first bugs](#good-first-bugs)

## Getting started

> To get started with git,first install git from the given link or if you have already installed it then update to its latest version.

Reviewer: ```suggestion
To get started with git, first install git from the given link or if you have already installed it then update to its latest version.
```

- [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)

After installing git successfully, make a account at github.

> - [Get started with GitHub](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account)

Reviewer: This links to help guide that tells the user to "Go to GitHub's Pricing page." and "Read the information about the different products and subscriptions that GitHub offers, then click the upgrade button under the subscription you'd like to choose.": we probably just want to send them to the "create free account" page to keep things simple.

## First contribution
Making a contribution is always good way to start in GitHub.
Let's make your first contribution using some simple steps and learning some commands.

> - [Make your contribution](https://github.com/firstcontributions/first-contributions)

Reviewer: I appreciate that we're linking out to external resources rather than creating our own.

## Good first bugs

> Now, you have learned how to make contribution then try some of our easy buge for you.

Reviewer: ```suggestion
Now, you have learned how to make contribution then try some of our easy bugs for you.
```

> - [good first](https://github.com/mozilla-mobile/shared-docs/contribute)

Reviewer: ```suggestion
- [Our tips for finding easy bugs](https://github.com/mozilla-mobile/shared-docs/blob/master/android/CONTRIBUTING_code.md#finding-issues-to-work-on)
- [fenix `good first issue`s](https://github.com/mozilla-mobile/fenix/labels/good%20first%20issue)
```

Reviewer: Changes I made:
- the `.../contribute` link appears to have more than just `good first issue` so I wanted to specifically link to those
- Fenix is our main project so I changed the repo from `shared-docs` to Fenix
- I also linked to existing documentation we have on finding good first issues
